src/common/libutil/Makefile.am

```diff
@@ -100,7 +100,9 @@ libutil_la_SOURCES = \
 	strstrip.c \
 	strstrip.h \
```

Comment:
Commit message for "add signal name/number helper routines" has the return value as `int` for `signutil_signame()`.  Should be `const char *`